README.rst

```diff
@@ -64,6 +64,13 @@ following RFC-4752, section 3.1:
 
     import winkerberos as kerberos
 
+    def get_channel_binding_struct(response):
+        # This is 'tls-server-end-point:' + certificate hash of the server
+        application_data = b'tls-server-end-point:D01402E0F16F30ED71B02B655AD71C7B0ADA73DE5FBD8134021A794FFA1EECE8'
```

Comment:
When testing this I just used a SHA256 hash as my server endpoint matched the RFC for this. I have done some extra work in requests-ntlm and requests-kerberos to support hashed > SHA256 by manually parsing the DER of the cert as it is just an ASN1 structure. An example of this can be found at https://github.com/jborean93/requests-kerberos/blob/3a3ef105e01952a61c993ac275555d45d8ad8101/requests_kerberos/kerberos_.py#L96-L168

I can definitely add some more info to the README for this though and give some more concrete examples.